Modules/Tokens/test/Tokens.API.Tests.Integration/Models/HttpResponse.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Net;
+
+namespace Tokens.API.Tests.Integration.Models;
+public class HttpResponse<T>
+{
+    public T Content { get; set; }
```

Comment:
I think you can use `ResponseContent<T>` here. You pass `ResponseContent` on every usage anyway, as far as I can see

```suggestion
    public ResponseContent<T> Content { get; set; }
```